turbo.json

```diff
@@ -15,7 +16,8 @@
                 "test:treeshakability:node",
                 "test:typecheck",
                 "test:unit:browser",
-                "test:unit:node"
+                "test:unit:node",
+                "start"
```

Comment:
I feel like the _semantics_ of what you're trying to do here is ‘test the examples.’ Instead of adding a new dimension (`start`) to the build, what would you think of adding a `test:unit:node` _task_ to each example (ie. that just ran `run:example`).